homeassistant/components/iperf3/__init__.py

```diff
@@ -66,6 +72,22 @@
         device_class=SensorDeviceClass.DATA_RATE,
         native_unit_of_measurement=UnitOfDataRate.MEGABITS_PER_SECOND,
     ),
+    SensorEntityDescription(
+        key=ATTR_JITTER,
+        name=ATTR_JITTER.capitalize(),
+        icon="mdi:timer",
```

Comment:
This has a device class, which provides the default icon

So! The icon can be removed